preseed.cfg

```diff
@@ -0,0 +1,75 @@
+# DOCUMENTATION
+# https://www.debian.org/releases/stretch/i386/apbs04.html.fr
+# https://www.debian.org/releases/squeeze/example-preseed.txt
+
+
+### Set hostname and domain ###
+d-i netcfg/get_hostname string debian
+d-i netcfg/get_domain string dumas-desktop
+
+# If you prefer to configure the network manually, uncomment this line and
+# the static network configuration below.
+d-i netcfg/disable_autoconfig boolean true
+
+# Static network configuration.
+d-i netcfg/get_nameservers string 192.168.122.1
+d-i netcfg/get_ipaddress string 192.168.122.42
+d-i netcfg/get_netmask string 255.255.255.0
+d-i netcfg/get_gateway string 192.168.122.1
+d-i netcfg/confirm_static boolean true
+
+### Set mirror ###
+d-i mirror/country string manual
+d-i mirror/http/hostname string http.us.debian.org
+d-i mirror/http/directory string /debian
+d-i mirror/http/proxy string
+
+### Clock ###
+d-i clock-setup/utc boolean true
+d-i time/zone string Europe/Paris
+
+### Locales ####
+d-i debian-installer/locale select en_US
+d-i console-setup/ask_detect boolean false
+d-i keyboard-configuration/xkb-keymap select fr(latin9)
+
+### Users ###
+d-i passwd/root-login boolean false
+d-i passwd/user-fullname string dumasg
+d-i passwd/username string dumasg
+d-i passwd/user-password-crypted password $6$lU4vleehvGkCNy2d$/omyJSLW51SuasR5Xw2v2EneqaTr9Js4q1ntk5PuLzkapZEJsscqgQ/0S/jONN7.d2vdy9DoO/riB9J5TAvka.
+d-i passwd/user-uid string 1010
```

Comment:
OK, mais pourquoi?

Reply:
Cette ligne vient de l'exemple du preseed de la doc de debian
https://www.debian.org/releases/bookworm/example-preseed.txt

# Create the first user with the specified UID instead of the default.
#d-i passwd/user-uid string 1010